src/main/scala/mesosphere/marathon/health/HealthCheckWorkerActor.scala

```diff
@@ -73,6 +73,13 @@ class HealthCheckWorkerActor extends Actor with ActorLogging {
       }
     }
 
+  def getPort(task: MarathonTask, check: HealthCheck): Option[Integer] = {
```

Comment:
this method can be `private[this]` or embedded in the `doCheck` method. What do you think?